extensions/monorepo-manager/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "monorepo-manager",
+  "version": "0.0.1",
+  "title": "Manage Monorepo Projects/Workspaces",
+  "description": "A Raycast extension helps to manage monorepo projects/workspaces in your machine. It also list all packages in each monorepo project/workspace",
+  "icon": "command-icon.png",
+  "author": "tung-dang",
+  "categories": [
+    "Productivity",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "manageMonorepo",
+      "title": "Manage Monorepo",
+      "description": "Manage packages/workspaces",
+      "mode": "view",
+      "icon": "command-icon.png"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "rootSourceFolder",
+      "title": "Root Source Folder Path",
+      "description": "A absolute folder path contains all workspaces/projects",
+      "type": "directory",
+      "required": true,
+      "placeholder": "/Uers/your-name/sources/",
+      "default": ""
+    },
+    {
+      "name": "cacheDuration",
+      "title": "Cache Duration",
+      "description": "Cache duration (ms). Default is 1 day",
+      "type": "textfield",
+      "required": false,
+      "placeholder": "86400000",
+      "default": "86400000"
+    },
+    {
+      "name": "slackRedirectLink",
+      "title": "Slack Channel Redirection Link",
+      "description": "A Slack link is used to redirect to a specific channel, which is gotten from package information",
+      "type": "textfield",
+      "required": false,
+      "placeholder": "https://<your-workspace>.slack.com/app_redirect?channel={query}",
+      "default": ""
+    },
+    {
+      "name": "mainFieldName",
+      "title": "Field in `package.json` File Containing Detail of a Package",
+      "description": "If the field name is not specified, some detail info of current package will be omitted",
+      "type": "textfield",
+      "required": false,
+      "default": ""
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "latest",
+    "@yarnpkg/core": "latest",
+    "fs-extra": "latest",
+    "lodash": "latest",
+    "parse-git-config": "latest"
```

Comment:
You should lock dependencies to a particular version, otherwise we might end up in weird egde cases based on installation time 😊

Reply:
I will make the fix now. Thanks!